cypress/integration/plp.test.js

```diff
@@ -45,11 +60,13 @@ describe('Search page Filters and Sorting options', () => {
     cy.visit(pages.collection, options)
     cy.waitForHydration()
 
+    const priceId = '[data-testid="store-card"] [data-testid="price"]'
+
     cy.getById('search-sort')
       .should('exist')
       .select('price_asc')
       .then(() => {
-        cy.getById('price').should(($prices) => {
+        cy.get(priceId).should(($prices) => {
```

Comment:
Why does get by id don't work?

Reply:
It's because we added a "You might also like" section to the bottom of the PLP with a list of 5 products. This section is rendered **before** the product grid is loaded, so the prices for these 5 products were being selected! Since they aren't part of the product grid, they cannot be sorted, and were breaking the tests till we could better scope this selector.

![CleanShot 2022-01-27 at 14 43 01](https://user-images.githubusercontent.com/381395/151414165-721f5bd2-7b3b-458f-8e5a-7a9c60700b8f.png)